src/Slider/Thumbs.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/prop-types */
+import React, { useRef } from "react";
+import { VisuallyHidden, mergeProps, useFocusRing, useSliderThumb } from "react-aria";
+import cc from "classcat";
+
+const getRailWidth = (lowerInputThumbProps, higherInputThumbProps) => parseFloat(higherInputThumbProps.style?.left.toString().slice(0, -1)) - parseFloat(lowerInputThumbProps.style?.left.toString().slice(0, -1));
+
+const useThumb = (index, trackRef, name, state) => {
+  const inputRef = useRef(null);
+  const { thumbProps, inputProps } = useSliderThumb({
+    index,
+    trackRef,
+    inputRef,
+    name
+  }, state);
+  const { focusProps, isFocusVisible } = useFocusRing();
+  return {
+    inputRef,
+    thumbProps,
+    inputProps,
+    focusProps,
+    isFocusVisible
+  }
+}
+
+const Thumb = ({ thumbProps, inputRef, inputProps, focusProps, isFocusVisible, ariaLabel }) => {
+  return (
+    <div
+      {...thumbProps}
+      className={cc(["thumb", { "focus": isFocusVisible }])}
+    >
+      <VisuallyHidden>
+        <input ref={inputRef} {...mergeProps(inputProps, focusProps)} aria-label={ariaLabel} />
```

Comment:
I think this is where we want the `onChange` to come from - `Thumbs` can fire `onChange` with the value of both inputs, and then `Slider` will be able to surface that to consumers